lab_2_tokenize_by_bpe/target_score.txt

```diff
@@ -1 +1 @@
-0
+6
```

Comment:
only 6?(

Reply:
Didn't have enough time to do all the tasks and make my PR green in time, so decided to leave only 6 :(